downloadGeo.sh

```diff
@@ -0,0 +1,6 @@
+curl -O geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz 
+curl -O geolite.maxmind.com/download/geoip/database/GeoLite2-Country.mmdb.gz 
+gunzip -c GeoLite2-City.mmdb.gz > _private/GeoLite2-City.mmdb
+gunzip -c GeoLite2-Country.mmdb.gz > _private/GeoLite2-Country.mmdb
+rm GeoLite2*
```

Comment:
This is a total edge case but I think it would be a tiny bit safer to explicitly delete the two files you want to delete rather than putting a star there. Worst case scenarios include 1) someone is working on a GeoLite2 feature and wants to keep certain files around (at least temporarily) and then this deletes them and the user doesn't know where they went and why. Or 2) someone somehow accidentally puts a space in between the `GeoLite2` and the `*` and this line deletes `GeoLite2` and all files in the directory.

`rm GeoLite2-City.mmdb.gz` and `rm GeoLite2-Country.mmdb.gz` would make me feel a little bit better.

FWIW Luc disagrees with me that this is a problem.